tbx/impact_reports/blocks.py

```diff
@@ -10,7 +10,7 @@
 
 
 class ImpactReportHeadingBlock(StructBlock):
-    image = ImageChooserBlock(required=False)
+    image = ImageChooserBlock(required=True)
```

Comment:
I have updated this because the current impact report always adds an image block, and because the anchor link is designed to land half way down the image - so if it isn't there, the anchor landing position will look off.